### ISSUE TITLE
Create maps using raster data from geoknife

### ISSUE THREAD

**Issue by lindsaycarr** [USGS-VIZLAB/how-to-gdp#35]
Dependent on #34 getting merged.

This affects two figures that should share a lot of functions because they really only differ by the geometry input. The figure showing the data layer as an input (using area surrounding the watershed for context) + the clipped version of the data to just the watershed (in gridded form before it's averaged to one value for the polygon).

Fetch ids: `yahara_precip` and `context_precip`
Process ids: `yahara_precip_timestep` and `context_precip_timestep` (note that these currently share a processor with the non-raster data)
Visualize ids: `inputs_data` and `yahara_precip_clip`
Publish ids: `publish_inputs_data` and `publish_yahara_precip`

I commented out the parts of the viz.yaml that this pertains to since it broke the make process on fetch with this error from geoknife:

```
geoknife::check(job)
$status
[1] "GeoTiff Generation Error: GeoTiffUtilException Trace: Calling sequence (top to bottom)\n\n--level 1--------------------------------------\nClass/Method: GeoTiffUtils/generateGeoTiffZipFromGrid\nId          : GeoTiffUtilException : NetCDF GeotiffWriter exception.\nMessage     : Unable to generate Tiff image from grid [dods://cida.usgs.gov/thredds/dodsC/stageiv_combined].  Exception: Must have 1D x and y axes for Total_precipitation_surface_1_Hour_Accumulation"

$URL
NULL

$statusType
[1] "ProcessFailed"
```

Seems like this is from using "subset" as the knife and "geotiff" as the output.

**Comment by wdwatkins** [comment]
Should anything else be on these maps besides the raster?

**Comment by lindsaycarr** [comment]
The only thing is working out the legend to indicate what all the colors mean.

**Comment by lindsaycarr** [comment]
Closed by #47
